src/client/helpers/entity.tsx

```diff
@@ -203,11 +203,14 @@ export function getEditionPublishers(edition) {
 
 	if (hasPublishers) {
 		return edition.publisherSet.publishers.map(
-			(publisher) => (
-				<a href={`/publisher/${publisher.bbid}`} key={publisher.bbid}>
-					{_get(publisher, 'defaultAlias.name', publisher.bbid)}
-				</a>
-			)
+			(publisher) =>
+				(
+					<span key={publisher.bbid}>
+						<a href={`/publisher/${publisher.bbid}`}>
+							{_get(publisher, 'defaultAlias.name', publisher.bbid)}
+						</a> &nbsp;
```

Comment:
It would be good to separate the Publishers with comas, currently it's hard to know they are different entities:
<img width="205" alt="image" src="https://user-images.githubusercontent.com/6179856/172901008-a71871a4-2011-4e81-8449-04e5d5e7c8b1.png">